hooks/getFeatures.ts

```diff
@@ -0,0 +1,29 @@
+import { Manifest } from "@hyperion-framework/types";
```

Comment:
This is some tech debt here. I'm not even sure how `@hyperion-framework/types` is still in our stack.

We should do a `npm install @iiif/presentation-3` and then replace this:

```suggestion
import { Manifest } from "@iiif/presentation-3";
```

Reply:
👍🏻